cfgov/v1/management/commands/delete_page.py

```diff
@@ -0,0 +1,24 @@
+from django.core.management.base import BaseCommand, CommandError
+
+from v1.models import CFGOVPage
+
+
+class Command(BaseCommand):
+    help = 'Deletes a page by its slug name'
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--slug',
```

Comment:
There has to be a way to use the `route()` method on pages so we can use a path and not the primary key.

```
>>> from django.http import HttpRequest
>>> from wagtail.wagtailcore.models import Page
>>> request = HttpRequest()
>>> request.is_staging = True
>>> root = Page.objects.get(slug='cfgov')
>>> path = "/policy-compliance/enforcement/actions/"
>>> path_components = [component for component in path.split('/') if component]
>>> page, args, kwargs = root.route(request, path_components)
>>> page
<BrowseFilterablePage: Enforcement Actions>
```

(Based on [how Wagtail resolves paths](https://github.com/wagtail/wagtail/blob/master/wagtail/wagtailcore/views.py#L19-L20))